package.json

```diff
@@ -321,9 +321,10 @@
     },
     "dependencies": {
         "@azure/arm-resources": "^3.0.0",
+        "@azure/arm-resources-profile-2020-09-01-hybrid": "^1.0.0",
```

Comment:
Yeah why is Storage on `2019-03-01-hybrid`, but this is on `2020-09-01-hybrid`? I would think we want to be consistent

Reply:
Update storage to profile-2020-09-01-hybrid, make another PR https://github.com/microsoft/vscode-azurestorage/pull/953.